### DISCUSSION TITLE
Error: Can not run generator since the current folder does not appear to be a Feathers application. Either your package.json is missing or it does not have `feathers` property.

### DISCUSSION THREAD

**Discussion by SharpBCD** [feathersjs/feathers#3302] [Q&A] [answered]
### Steps to reproduce

Existing feathers app, here are the dependencies:
```
"dependencies": {
    "@feathersjs/authentication": "^4.5.15",
    "@feathersjs/authentication-local": "^4.5.15",
    "@feathersjs/authentication-oauth": "^4.5.15",
    "@feathersjs/configuration": "^4.5.15",
    "@feathersjs/errors": "^4.5.15",
    "@feathersjs/express": "^4.5.15",
    "@feathersjs/feathers": "^4.5.15",
    "@feathersjs/socketio": "^4.5.15",
    "@feathersjs/transport-commons": "^4.5.15",
    "compression": "^1.7.4",
    "cors": "^2.8.5",
    "feathers-knex": "^8.0.1",
    "firebase-admin": "^11.0.1",
    "helmet": "^5.1.0",
    "knex": "^2.3.0",
    "pg": "^8.8.0",
    "serve-favicon": "^2.5.0",
    "winston": "^3.8.2",
    "winston-daily-rotate-file": "^4.7.1",
    "winston-transport": "^4.5.0"
  },
```
feathers cli version is 5.0.8

- When running 'feathers generate service' I got the error: "Error: Can not run generator since the current folder does not appear to be a Feathers application.
Either your package.json is missing or it does not have `feathers` property.
"

Tell us about the applicable parts of your setup.

**Module versions** (especially the part that's not working):

**NodeJS version**:
Node v18.17.1
**Operating System**:
Windows
**Browser Version**:
Chrome latest
**React Native Version**:

**Module Loader**:
yarn

**Comment by daffl** [comment, marked as answer]
The v5 CLI does not work with v4 applications. You need to install `@feathersjs/cli@4`

**Reply by SharpBCD** [reply]
I confirm. Downgrading to V4 and everything worked just fine.

**Reply by MurilloKerr** [reply]
I ran the command "npm install @feathersjs/cli@4" but I still have the problem.
![imagem_2023-12-19_184803415](https://github.com/feathersjs/feathers/assets/92184259/e1eca38b-47e3-43e6-a6b0-d87faf01553b)

**Reply by MurilloKerr** [reply]
Solved! Need to downgrade NodeJS and NPM too.
